develop-guide/writeblitz/writeblitz.rst

```diff
@@ -550,6 +569,41 @@ Action ``compare`` allows you to verify the values of the saved variables. Below
         - "'%VARIABLES{os}' == 'NX-OS' and '%VARIABLES{date_created}' == '10/22/2019 10:00:00 [10/22/2019 16:57:31]'"
         - " %VARIABLES{bootflash} >= 290000 or '%VARIABLES{bios}' == '07.33'"
 
+
+
+.. note::
+
+    The starting message of an step in etrade logs, is predefined. However, that value can be changed.
+    In your action you can specify your custom message like the example below and change the starting action log message.
+
+.. code-block:: YAML
+
+    - execute:
+        command: show version
+        device: PE1
+        custom_step_message: This is a test to see if a custom_step_message would be applied
+        timeout: 100
+
+as shown in the image you can see how in the logs the starting message is customized.
+
+.. image:: ../images/custom_step_msg.png
+   :width: 200%
+
+.. note::
+
+    You can get a Passed result for an action that is expected to fail by using setting the key; ``expected_failure: True``.
+
+.. code-block:: YAML
+
+    # The command doesnt exist so action should error out but since it was anticipated that the command wouldn't work.
```

Comment:
does this work for error too? Not just for failure?

Reply:
Errors that we are handling yes. but if the user is missing to put the device in the action. No its not supported for that. Should we support that ?! no right?

Reply:
In my mind, it should only for failure  not error

Reply:
There is only one error, that we are handeling that I set this to pass, which is for api error, on function name or args. I thought in case they have doubts about the function name maybe but they want to give it try. Just above that there is an error, that I didnt apply this to
![image](https://user-images.githubusercontent.com/60662436/94506086-976e0400-01da-11eb-9df2-25a18837bd0c.png)